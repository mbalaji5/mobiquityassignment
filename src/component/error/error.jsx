import React from "react";
const Error = () => {
  return <div>Service "ergast.com" is down.Please try again some time!</div>;
};
export default Error;
