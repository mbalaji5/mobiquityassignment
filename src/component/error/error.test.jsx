import { render, screen } from "@testing-library/react";
import { Error } from "../error";
describe("should validate error component", () => {
  it("validate error component text", () => {
    render(<Error />);
    const element = screen.getByText(
      /Service "ergast.com" is down.Please try again some time!/i
    );
    expect(element).toBeDefined();
  });
});

describe("Should validate error detail", () => {
  it("Should validate shopshot", () => {
    const { baseElement } = render(<Error />);
    expect(baseElement).toMatchSnapshot();
  });
});
