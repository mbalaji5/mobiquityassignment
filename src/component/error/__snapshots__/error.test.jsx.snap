// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should validate champion detail Should validate shopshot 1`] = `
<body>
  <div>
    <div>
      Service "ergast.com" is down.Please try again some time!
    </div>
  </div>
</body>
`;

exports[`Should validate error detail Should validate shopshot 1`] = `
<body>
  <div>
    <div>
      Service "ergast.com" is down.Please try again some time!
    </div>
  </div>
</body>
`;
